Makefile

```diff
@@ -53,8 +54,8 @@ FELIX_GPL_SOURCE=filesystem/included-source/felix-ebpf-gpl.tar.gz
 INCLUDED_SOURCE=$(BIRD_SOURCE) $(FELIX_GPL_SOURCE)
 
 # Versions and locations of dependencies used in tests.
-CALICOCTL_VER?=master
-CNI_VER?=master
+CALICOCTL_VER?=v3.14.1
```

Comment:
These should be `master` still, since this is the Makefile for the master branch.

Reply:
Agree, the specific v3.14.1 thing is just the tag my special case needs when the latest stable is v3.14.1. A universal match to `master` should be our goal.